boards/wsn430-v1_3b/include/board.h

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (C) 2018 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_wsn430-v1_3b WSN430 v1.3b
+ * @ingroup     boards
+ * @brief       Support for the Senslab WSN430 v1.3b board
+ *
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the Senslab WSN430 v1.3b board
+ *
+ * @author      Martine Lenders <m.lenders@fu-berlin.de>
+ */
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "board_common.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Definition of the interface to the CC1101 radio
+ */
+#define CC110X_PARAMS_BOARD     {   .spi  = SPI_DEV(0),     \
+                                    .cs   = GPIO_PIN(P4, 1), \
+                                    .gdo0 = GPIO_PIN(P1, 3), \
+                                    .gdo1 = GPIO_PIN(P5, 2), \
+                                    .gdo2 = GPIO_PIN(P1, 4) }
```

Comment:
I've taken those from the [WSN430 1.3b schematics](https://github.com/iot-lab/iot-lab/wiki/Hardware_Wsn430-node#schematics-and-datasheets).